.github/workflows/apk.yml

```diff
@@ -23,39 +23,43 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v3
 
-        # Load google-services.json and local.properties from the secrets
-      - name: Decode secrets
+      - name: Set up JDK
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'zulu'
+          java-version: '17'
```

Comment:
I think it's best to stay consistent with our `build.gradle.kts`, however I'm not sure if it's the same as the version specified in the image below :

![image](https://github.com/user-attachments/assets/72738531-17f4-4f9f-a48a-a51cf4baca82)

Reply:
According to my own research, as long as we've got the sourceCompatibility and targetCompatibility set to 11, it will only compile to Java 11 language features. Keeping it to 17 is purely arbitrary and because of the previous workflow definition. I see no need to change